model/lifecycle.go

```diff
@@ -341,8 +341,8 @@ func RestartVersion(versionId string, taskIds []string, abortInProgress bool, ca
 	if err != nil {
 		return errors.WithStack(err)
 	}
-	// remove execution tasks in case the caller passed both display and execution tasks
-	// the functions below are expected to work if just the display task is passed
+	// Remove execution tasks in case the caller passed both display and execution tasks.
```

Comment:
Not actually PR related, but should we add capitalization of comments to our style guide?

Reply:
It is already in the style guide but is listed as preferable where I actually thought it was required https://docs.google.com/document/d/1-DB1FQ_zb5fX_0pnVcwyU0HmJPrloFBu6qSE8dkk8PU/edit# ; I think this was done to avoid PRs from needing to be nit picky (oops) / to avoid us needing to go through the codebase explicitly to fix everything (like with error messages)